packages/cashier/src/stores/payment-agent-store.ts

```diff
@@ -93,29 +108,28 @@ export default class PaymentAgentStore {
     }
 
     @action.bound
-    setList(pa_list) {
+    setList(pa_list: TPartialPaymentAgentList): void {
         this.list.push(pa_list);
     }
 
     @action.bound
-    clearList() {
-        this.list = [];
+    clearList(): void {
+        this.list = [] as unknown as IObservableArray<TPartialPaymentAgentList>;
     }
 
     @action.bound
-    async setPaymentAgentList(pa_list) {
```

Comment:
remove `pa_list` parameter from setPaymentAgentList because we don't call the function with this parameter in our code